requirements-docs.txt

```diff
@@ -1,3 +1,7 @@
 mkdocs-material==9.4.6
```

Comment:
Upgrade to `9.5.5`? it seems to have some nices fixes compared to this version.

https://squidfunk.github.io/mkdocs-material/changelog/